auth-web/src/routes/router.ts

```diff
@@ -47,6 +47,13 @@ function mapReturnPayVars (route: any) {
   }
 }
 
+// Feature Flagging HomeView Components
+// For Development only
```

Comment:
Not for development only. Actually the opposite :)
But definitely temporary while leading up to MVP.

Reply:
That is a good point :) My thinking is backwards!